overlord/snapstate/backend.go

```diff
@@ -68,7 +68,7 @@ type managerBackend interface {
 	CopySnapData(newSnap, oldSnap *snap.Info, meter progress.Meter) error
 	LinkSnap(info *snap.Info, dev boot.Device, linkCtx backend.LinkContext, tm timings.Measurer) (rebootRequired bool, err error)
 	StartServices(svcs []*snap.AppInfo, meter progress.Meter, tm timings.Measurer) error
-	StopServices(svcs []*snap.AppInfo, reason snap.ServiceStopReason, meter progress.Meter, tm timings.Measurer) error
+	StopServices(svcs []*snap.AppInfo, alsoDisable bool, reason snap.ServiceStopReason, meter progress.Meter, tm timings.Measurer) error
```

Comment:
I would strongly prefer flags to a boolean here. It's also just one argument but using it is much more readable.

Reply:
Done.